webui/react/src/types.ts

```diff
@@ -317,24 +317,28 @@ export interface ExperimentPagination extends WithPagination {
   experiments: ExperimentItem[];
 }
 
-export enum RunState {
-  Active = 'ACTIVE',
-  Paused = 'PAUSED',
-  StoppingCanceled = 'STOPPING_CANCELED',
-  Canceled = 'CANCELED',
-  StoppingCompleted = 'STOPPING_COMPLETED',
-  Completed = 'COMPLETED',
-  StoppingError = 'STOPPING_ERROR',
-  Errored = 'ERROR',
-  Deleted = 'DELETED',
-  Deleting = 'DELETING',
-  DeleteFailed = 'DELETE_FAILED',
-  Unspecified = 'UNSPECIFIED',
-  Queued = 'QUEUED',
-  Pulling = 'PULLING',
-  Starting = 'STARTING',
-  Running = 'RUNNING',
-}
+export const RunState = {
+  ACTIVE: 'ACTIVE',
+  CANCELED: 'CANCELED',
+  COMPLETED: 'COMPLETED',
+  DELETE_FAILED: 'DELETE_FAILED',
+  DELETED: 'DELETED',
+  DELETING: 'DELETING',
+  ERROR: 'ERROR',
+  PAUSED: 'PAUSED',
+  PULLING: 'PULLING',
+  QUEUED: 'QUEUED',
+  RUNNING: 'RUNNING',
+  STARTING: 'STARTING',
+  STOPPING_CANCELED: 'STOPPING_CANCELED',
+  STOPPING_COMPLETED: 'STOPPING_COMPLETED',
+  STOPPING_ERROR: 'STOPPING_ERROR',
+  UNSPECIFIED: 'UNSPECIFIED',
```

Comment:
Just to avoid churn we should probably leave these as CapitalCase, sorry for capitalizing them in the example I gave you (I was just trying to hammer something together quickly).